docs/kvbc/img/RootHashes.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Nit: Maybe it makes sense to use a similar notation in the leaf as the one used in the text before, i.e. `h((h(k1) || h(v1)) ...` instead of `H(H(KV)...`, just to avoid confusion.

Reply:
I'm going to leave it for now just to save time.